src/admin-views/tribe-options-tickets.php

```diff
@@ -180,61 +200,67 @@
 		array( 'cmd' => '_profile-ipn-notify' ),
 		tribe( 'tickets.commerce.paypal.gateway' )->get_settings_url()
 	);
-	$ipn_notification_settings_link     = '<a href="' . $paypal_ipn_notify_url_setting_link . '" target="_blank">' . esc_html__( 'Profile and Settings > My selling tools > Instant Payment Notification > Update','event-tickets' ) . '</a>';
+	$ipn_notification_settings_link     = '<a href="' . $paypal_ipn_notify_url_setting_link . '" target="_blank">' . esc_html__( 'Profile and Settings > My selling tools > Instant Payment Notification > Update',
+			'event-tickets' ) . '</a>';
 
 	$paypal_ipn_notification_history_link = add_query_arg(
 		array( 'cmd' => '_display-ipns-history' ),
 		tribe( 'tickets.commerce.paypal.gateway' )->get_settings_url()
 	);
-	$ipn_notification_history_link = '<a href="' . $paypal_ipn_notification_history_link . '" target="_blank">' . esc_html__( 'Profile and Settings > My selling tools > Instant Payment Notification > IPN History Page','event-tickets' ) . '</a>';
+	$ipn_notification_history_link        = '<a href="' . $paypal_ipn_notification_history_link . '" target="_blank">' . esc_html__( 'Profile and Settings > My selling tools > Instant Payment Notification > IPN History Page',
```

Comment:
Can we break this on the concat `.` instead of where it's currently doing it? Also `esc_url`